sql/clients_monthly_v1.init.sql

```diff
@@ -0,0 +1,18 @@
+CREATE IF NOT EXISTS TABLE
+  clients_monthly_v1
```

Comment:
`clients_monthly` is an enticing name because it's simple, but I think it's too easy to make the wrong assumption about what this is. With `clients_daily` being aggregations per client per day, I think the most common assumption is going to be that this is aggregations per month.

Mark may have already vetoed some of these, but perhaps it deserves another look given the new nature of this table compared to clients_last_seen. Some options:

- clients_daily_last_28
- clients_last_28_days
- clients_daily_longitudinal
- clients_daily_windowed
- client_days

Reply:
I see your suggestions and raise you clients_daily_window28. I also like

- clients_daily_window_28_days
- clients_daily_over_preceding_28_days
- clients_daily_windowed
- clients_daily_last_28_days
- clients_daily_

Reply:
Clearly, this right choice here is `clients_daily++` ;)

I think `clients_daily_window28` is pretty good.